app/models/export.rb

```diff
@@ -82,6 +59,24 @@ def execute
     snap_application.pdf.try(:unlink)
   end
 
+  def logger
+    return @_logger if @_logger.present?
+
+    logger = ActiveSupport::Logger.new(STDOUT)
+    logger.level = Logger::DEBUG
+    logger.formatter = Rails.application.config.log_formatter
+    @_logger = ActiveSupport::TaggedLogging.new(logger)
```

Comment:
A nice pattern I've seen for multi-line memoization is to wrap things in a `begin`/`end` block to wrap everything up together:

```ruby
def logger
  @_logger ||= begin
    logger = ActiveSupport::Logger.new(STDOUT)
    logger.level = Logger::DEBUG
    logger.formatter = Rails.application.config.log_formatter
    ActiveSupport::TaggedLogging.new(logger)
  end
end
```